File: 05/5.rb
def main()
	i = 20
	while (! isDivByRange20(i))
		i += 20
	end
	puts i
end

def isDivByRange20(n)
	for i in 20.downto(2)
		if (n % i != 0)
			return false
		end
	end
	return true
end

if __FILE__ == $0
	main()
end

File: 05/5.java
class main  
{
    public static boolean isDivByRange20(int n)
    {
        for (int i = 20 ; i >= 2 ; i--)
        {
            if(n%i != 0)
                return false;
        }
        return true;
    }

    public static void main(String[] args) 
    {
		int i = 20;
		while (! isDivByRange20(i))
			i += 20;
		System.out.println(""+i);
    }
}
